### PR TITLE
Refactoring the sidebar

### PR DESCRIPTION
**Pull request by Duncid**
## Description

Changing the Sidebar:
- Moving Avatar & menu down the sidebar
- Moving help in the same zone
- Creating a visible button to hide sidebar

## Risk

UI breaks

## Deploy Plan
- Make Help menu dynamic (Porting Help modal specs) and functional (Trigger new conversation on click on question, insert @help and animate the input bar on click on @help button)

### CODE REVIEW

File: front/components/QuickStartGuide.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                        well.
+                      </p>
+                      <div className="space-y-2 text-sm">
+                        <p>
+                          Assistants in Dust can be provided by Dust, by your
+                          company (Company assistants) or by your coworkers
+                          (Shared assistants).
+                        </p>
+                        <p>
+                          Specialised assistants generally combine a model with
```
Comment (aubin-tchoi) [inline front/components/QuickStartGuide.tsx:84]: OOC: what's the policy between using US vs UK English? (applies to specialized and summarize in this file)

Comment (JulesBelveze) [reply front/components/QuickStartGuide.tsx:84]: 🤷🏼‍♂️

Comment (tdraier) [reply front/components/QuickStartGuide.tsx:84]: current code is using US , so should change here to `Specialized` and `summarize`

File: front/components/navigation/NavigationSidebar.tsx [hunk 2]
```diff
@@ -59,33 +61,18 @@ export const NavigationSidebar = React.forwardRef<
...
-          {user && <UserMenu user={user} owner={owner} />}
-        </div>
+        {user && user.workspaces.length > 1 ? (
+          <WorkspacePicker
+            user={user}
+            workspace={owner}
+            onWorkspaceUpdate={(workspace) => {
+              const assistantRoute = `/w/${workspace.sId}/assistant/new`;
+              if (workspace.id !== owner.id) {
+                void router.push(assistantRoute).then(() => router.reload());
```
Comment (tdraier) [inline front/components/navigation/NavigationSidebar.tsx:71]: please replace with await :pray:

**Review by tdraier — APPROVED** [review]

**Pull request merged**